src/components/elementView/elementTable.tsx

```diff
@@ -101,10 +103,11 @@ interface ElementTableState {
 }
 
 interface ElementTableProps {
-  isDecommissioned: boolean;
+  isDecommissioned: boolean; // TODO: remove this and use assetType instead
```

Comment:
Probably not going to do this because it's correct as is, just a little redundant

Reply:
ev4 amiright